saleor/static/dashboard/js/components/misc.js

```diff
@@ -53,6 +54,9 @@ $('#product-is-published').on('click', (e) => {
   }
 });
 
+// WYSIWYG editor
+const editor = new MediumEditor('#id_description');
```

Comment:
Shouldn't this rather be something like `'.rich-editor'`?

Reply:
It's still draft - next step is creating widget `RichTextWidget` or something like this